bert/colossalai_utils/bert_config_zerov1.json

```diff
@@ -0,0 +1,28 @@
+{
+    "method": "colossalai",
+    "model": {
+        "type": "bert_small"
+    },
+    "hyperparameter": {
+        "batch_size": 8,
+        "steps_per_epoch": 10
+    },
+    "fp16": {
+        "initial_scale": 32768,
+        "min_scale": 1,
+        "growth_factor": 2.0,
+        "backoff_factor": 0.5,
+        "growth_interval": 1000
+    },
+    "gradient_clipping": 0.0,
+    "zero": {
+        "mixed_precision": true, 
+        "reshard_after_forward": false, 
+        "offload_config" : {
+            "device": "cpu"
+        },
+        "version": 1
+    },
+    "use_mem_monitor": true,
+    "sharded_model_version": 1
```

Comment:
remove this line

Reply:
Necessary to my v1 config, I removed it from v2 config.
Done.